flutter_screen_recording/lib/flutter_screen_recording.dart

```diff
@@ -1,37 +1,55 @@
+import 'dart:html' hide Platform;
+
 import 'package:flutter/foundation.dart';
 import 'package:flutter_screen_recording_platform_interface/flutter_screen_recording_platform_interface.dart';
 import 'dart:async';
 import 'dart:io';
 import 'package:flutter_foreground_plugin/flutter_foreground_plugin.dart';
 
 class FlutterScreenRecording {
-  static Future<bool> startRecordScreen(String name,
-      {String titleNotification, String messageNotification}) async {
+  static Future<bool> startRecordScreen(
+    String name, {
+    String titleNotification,
+    String messageNotification,
+    Function(String) onStop,
+  }) async {
     await _maybeStartFGS(titleNotification, messageNotification);
     final bool start =
-        await FlutterScreenRecordingPlatform.instance.startRecordScreen(name);
+    await FlutterScreenRecordingPlatform.instance.startRecordScreen(
+      name,
+      onStop: onStop,
+    );
     return start;
   }
 
-  static Future<bool> startRecordScreenAndAudio(String name,
-      {String titleNotification, String messageNotification}) async {
+  static Future<bool> startRecordScreenAndAudio(String name, {
+    String titleNotification,
+    String messageNotification,
```

Comment:
Same above

Reply:
Above code come from package author, We don't use that code so we do not need to maintain that now.